extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -425,6 +432,18 @@ export class AmpStoryPage extends AMP.BaseElement {
     return Promise.all(mediaPromises);
   }
 
+  /**
+   * Finds embedded components in page and prepares them for their expanded view
+   * animation.
+   */
+  findAndPrepareEmbeddedComponents() {
+    EMBEDDED_COMPONENTS_SELECTORS.forEach(selector => {
```

Comment:
Agreed, and since this is the only place it's used, you can actually just store that as the constant instead, so that it's not reconstructed every time.